index.html

```diff
@@ -192,14 +197,14 @@ <h3>Upgrade the app</h3>
             </li>
             <li>Install the app using <a href="https://pip.pypa.io/en/stable/">pip</a>:<br>
               <code style="width:650px">
-                $ pip install omero-figure
+                $ pip install -U omero-figure
```

Comment:
I think we can remove this whole section of "upgrade from a version of OMERO.figure prior to 2.0.1" since that is quite old now.